webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: path.join(__dirname, 'src', 'index.js'),
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'index.bundle.js',
+  },
+  mode: process.env.NODE_ENV || 'development',
+  resolve: {
+    modules: [path.resolve(__dirname, 'src'), 'node_modules'],
+    extensions: ['.js', '.jsx'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: ['babel-loader'],
+      },
+      {
+        test: /\.(css|scss)$/,
+        use: ['style-loader', 'css-loader'],
+      },
+      {
+        test: /\.(jpg|jpeg|png|gif|mp3|svg)$/,
+        use: ['file-loader'],
+      },
+    ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
```

Comment:
the config itself is very simple and doesn't comply with home task description
you need to have separate configurations for PROD and DEV envs, with prod version having more optimisation plugins and dev - dev tools plugins